Monika After Story/game/script-ch30.rpy

```diff
@@ -1270,6 +1281,11 @@ label ch30_loop:
     if mas_idle_mailbox.get_skipmidloopeval():
         jump ch30_post_mid_loop_eval
 
+    #Do the weather thing
+    if mas_weather.weatherProgress() and mas_isMoniNormal(higher=True):
```

Comment:
this will cause issues if we were meant to skip visuals. Think this should be moved to be part of the if statement for spaceroom.